eclair-core/src/main/scala/fr/acinq/eclair/wire/protocol/OfferCodecs.scala

```diff
@@ -19,135 +19,159 @@ package fr.acinq.eclair.wire.protocol
+    .typecase(UInt64(88), invoiceRequestPayerId)
+    .typecase(UInt64(89), invoiceRequestPayerNote)
+    .typecase(UInt64(240), signature)
+  ).complete
 
-  private val paymentInfo: Codec[PaymentInfo] = (("fee_base_msat" | millisatoshi32) ::
-    ("fee_proportional_millionths" | uint32) ::
-    ("cltv_expiry_delta" | cltvExpiryDelta) ::
-    ("htlc_minimum_msat" | millisatoshi) ::
-    ("htlc_maximum_msat" | millisatoshi) ::
-    ("features" | lengthPrefixedFeaturesCodec)).as[PaymentInfo]
+  private val invoicePaths: Codec[InvoicePaths] = tlvField(list(pathCodec).xmap[Seq[BlindedRoute]](_.toSeq, _.toList))
 
-  private val paymentPathsInfo: Codec[PaymentPathsInfo] = tlvField(list(paymentInfo).xmap[Seq[PaymentInfo]](_.toSeq, _.toList).as[PaymentPathsInfo])
+  private val paymentInfo: Codec[PaymentInfo] =
+    (("fee_base_msat" | millisatoshi32) ::
+      ("fee_proportional_millionths" | uint32) ::
+      ("cltv_expiry_delta" | cltvExpiryDelta) ::
+      ("htlc_minimum_msat" | millisatoshi) ::
+      ("htlc_maximum_msat" | millisatoshi) ::
+      ("features" | lengthPrefixedFeaturesCodec)).as[PaymentInfo]
 
-  private val paymentPathsCapacities: Codec[PaymentPathsCapacities] = tlvField(list(millisatoshi).xmap[Seq[MilliSatoshi]](_.toSeq, _.toList).as[PaymentPathsCapacities])
+  private val invoiceBlindedPay: Codec[InvoiceBlindedPay] = tlvField(list(paymentInfo).xmap[Seq[PaymentInfo]](_.toSeq, _.toList))
 
-  private val createdAt: Codec[CreatedAt] = tlvField(tu64overflow.as[TimestampSecond].as[CreatedAt])
+  private val invoiceCreatedAt: Codec[InvoiceCreatedAt] = tlvField(tu64overflow.as[TimestampSecond])
 
-  private val paymentHash: Codec[PaymentHash] = tlvField(bytes32.as[PaymentHash])
+  private val invoiceRelativeExpiry: Codec[InvoiceRelativeExpiry] = tlvField(tu32)
 
-  private val relativeExpiry: Codec[RelativeExpiry] = tlvField(tu32.as[RelativeExpiry])
+  private val invoicePaymentHash: Codec[InvoicePaymentHash] = tlvField(bytes32)
 
-  private val cltv: Codec[Cltv] = tlvField(uint16.as[CltvExpiryDelta].as[Cltv])
+  private val invoiceAmount: Codec[InvoiceAmount] = tlvField(tmillisatoshi)
 
-  private val fallbackAddress: Codec[FallbackAddress] = variableSizeBytesLong(varintoverflow, ("version" | byte) :: ("address" | variableSizeBytes(uint16, bytes))).as[FallbackAddress]
+  private val fallbackAddress: Codec[FallbackAddress] = (("version" | byte) :: ("address" | variableSizeBytes(uint16, bytes))).as[FallbackAddress]
 
-  private val fallbacks: Codec[Fallbacks] = tlvField(list(fallbackAddress).xmap[Seq[FallbackAddress]](_.toSeq, _.toList).as[Fallbacks])
+  private val invoiceFallbacks: Codec[InvoiceFallbacks] = tlvField(list(fallbackAddress).xmap[Seq[FallbackAddress]](_.toSeq, _.toList))
 
-  private val refundSignature: Codec[RefundSignature] = tlvField(bytes64.as[RefundSignature])
+  private val invoiceFeatures: Codec[InvoiceFeatures] = tlvField(featuresCodec)
+
+  private val invoiceNodeId: Codec[InvoiceNodeId] = tlvField(publicKey)
```

Comment:
This should be replaced by `invoiceCode`

Reply:
No, it's been removed in https://github.com/rustyrussell/lightning-rfc/pull/13